userguide/content/en/docs/adding-content/shortcodes/index.md

```diff
@@ -691,6 +691,20 @@ Alternately, Hugo you can display a message on the rendered page instead of thro
 {{</* readfile file="includes/deploy.yaml" code="true" lang="yaml" draft="true" */>}}
 ```
 
+### Include remote markdown files
+
+The `include-remote-md` shortcode fetches a remote markdown file and includes it in the page. This happens only when your Docsy project is built, so future changes of the remote markdown file are only included when you rebuild your project.
+
+If the file has a frontmatter, define the marker of the frontmatter in the second parameter of the shortcode (defaults to `---`). Note that if the content after the frontmatter includes the marker (for example, includes `---` as part of a markdown-formatted table) the content is truncated.
+
```

Comment:
```suggestion

If the file has a H1 (`# ....`) instead of frontmatter, you can put the title line in as the second parameter and it will be removed instead. For example:

{{</* include-remote-md "https://raw.githubusercontent.com/google/docsy/main/README.md" "# Docsy" */>}}
```